DEVELOPMENT.md

```diff
@@ -0,0 +1,33 @@
+# Setting up a Development Environment
+
+This document details how to set up a local development environment that will
+allow you to contribute changes to the project.
+
+Acquire sources and create virtualenv.
+```shell
+git clone https://github.com/langchain-ai/langchain-postgres
+cd langchain-postgres
+uv venv --python=3.13
+source .venv/bin/activate
+```
+
+Install package, editable.
+```shell
+poetry install --with dev,test,lint
+pip install greenlet
```

Comment:
curious why is greenlet required?

Reply:
and if required should it just be in pyproject.toml?

Reply:
Hi @eyurtsev, thanks for your reply

a) I will dig out again why greenlet is required, to check if I have not been misguided.
b) True. I will add it using another PR if it's really needed.